### PR TITLE
Added Slovak translation

### PR DESCRIPTION
**Pull request by LadislavM**

### CODE REVIEW

File: src/utils/initI18n.js [hunk 1]
```diff
@@ -2,14 +2,18 @@ import i18next from 'i18next';
 import en from '../../locales/en';
 import de from '../../locales/de';
 import fr from '../../locales/fr';
+import sk from '../../locales/sk';
+
+}
```
Comment (ohheyitsdave) [inline src/utils/initI18n.js]: added by mistake?

File: src/components/Nav.svelte [hunk 2]
```diff
@@ -134,4 +134,5 @@
         </div>
     </nav>
 </div>
-
+<a href="{`sk/${pathWithoutLang}`}" class="uppercase block md:pr-4"
+               class:selected="{lang === 'sk' ? 'selected' : ''}">{flag('sk')}</a>
```
Comment (ohheyitsdave) [inline src/components/Nav.svelte]: This should be in the src/component/LanguageSwitcher.svelte, else it's floating somewhere on the site

**Review by ohheyitsdave — CHANGES_REQUESTED** [review]
Hi @LadislavM, can you please fix the two small things and then it should be fine.
Thanks for your help  🙃

File: src/utils/initI18n.js (hunk 1 above)
Comment (LadislavM) [reply src/utils/initI18n.js]: yes :(

File: src/components/Nav.svelte (hunk 2 above)
Comment (LadislavM) [reply src/components/Nav.svelte]: Repaired. Switcher placed in src/component/LanguageSwitcher.svelte

**Review by ohheyitsdave — APPROVED** [review]
looks good. Will merge into develop-branch, test it on our staging environment and should be live soon. Thanks for your help  😊

**Pull request merged**